haskeline.cabal

```diff
@@ -20,7 +20,7 @@ Homepage:       http://trac.haskell.org/haskeline
 Bug-Reports:    https://github.com/judah/haskeline/issues
 Stability:      Stable
 Build-Type:     Custom
-extra-source-files: examples/Test.hs Changelog
+extra-source-files: examples/Test.hs Changelog includes/*.h
```

Comment:
Can you explain why this change was needed (vs how the existing header win_console.h is treated)?